#goal definition
string action
string navstrategy
bool use_inter_markers
string make_plan_mode
bool side_door_crossing
string node
geometry_msgs/Pose goal_pose

---
#result definition
int32 result
---
#feedback
int32 feedback
